cmake/dependencies/PICSAR.cmake

```diff
@@ -4,20 +4,36 @@ function(find_picsar)
         include(FetchContent)
         set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
 
-        # FIXME no option to control WarpX_QED_TABLE_GEN / Boost trigger yet
-
         FetchContent_Declare(fetchedpicsar
             GIT_REPOSITORY ${WarpX_picsar_repo}
             GIT_TAG        ${WarpX_picsar_branch}
             BUILD_IN_SOURCE 0
         )
         FetchContent_GetProperties(fetchedpicsar)
+        
+        # Enable or disable QED lookup tables generation
+        # If table generation is enabled, enable or disable 
+        # openMP support depending on WarpX_COMPUTE
+        if(WarpX_QED_TABLE_GEN)        
+            set(PXRMP_QED_TABLEGEN ON CACHE INTERNAL "")
+            if(WarpX_COMPUTE STREQUAL OMP)        
+                set(PXRMP_QED_OMP ON CACHE INTERNAL "")
+            else()
+                set(PXRMP_QED_OMP OFF CACHE INTERNAL "")
+            endif()
+        else()
+            set(PXRMP_QED_TABLEGEN OFF CACHE INTERNAL "")
+            set(PXRMP_QED_OMP OFF CACHE INTERNAL "")
+        endif()
+        
+        # Always disable tests
+        set (PXRMP_QED_TEST OFF CACHE INTERNAL "")
```

Comment:
This should move above the `FetchContent_Declare` call :)

Reply:
Ah right! I've just done that!